katas/02KataTwo.js

```diff
@@ -5,11 +5,18 @@ var assert = require('assert');
 // Given an array A, find the int that appears an odd number of times.
 // There will always be only one integer that appears an odd number of times.
 
+var count = 0;
 function findOdd(arr) {
     // Your code here
+    for (var i = 0; i<=arr.length; i++){
```

Comment:
I am still working on Kata02. I pushed it by accident.

On Sun, Jun 5, 2016 at 1:36 PM, Chen Yang <notifications@github.com> wrote:

> In katas/02KataTwo.js
> <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/47#discussion_r65822048>
> :
>
> >  function findOdd(arr) {
> >      // Your code here
> > +    for (var i = 0; i<=arr.length; i++){
>
> Do you have any questions about kata02? We can go over this in next lesson.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/47/files/870b3567abf267f9feea36d285edd90e5683b3ab..98a9670d57feeb14821948a9dfaca09bb2a25b7b#r65822048>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQNZVJKSBa9z-KCQeTqpsPDVpsb42Bxpks5qIxdEgaJpZM4Ie4rG>
> .
>